flink-connector-mongodb-e2e-tests/pom.xml

```diff
@@ -65,6 +65,11 @@ under the License.
 			<artifactId>mongodb</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
```

Comment:
Hi @GOODBOY008, why do we need the `commons-codec` dependency?

Reply:
Yes, before add this dependency https://github.com/GOODBOY008/flink-connector-mongodb/actions/runs/8557577952/job/23450146047#step:15:11104 ,ci failed with error.

Reply:
@GOODBOY008 that seems a mistake, see https://issues.apache.org/jira/browse/COMPRESS-659.
I think we should bump commons-compress version to 1.26.1.